website/integrations/services/semaphore/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Integrate with Semaphore UI
+sidebar_label: semaphore
+---
+
+# Semaphore UI
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Semaphore UI
+
+> Semaphore UI is a modern web interface for managing popular DevOps tools.
+> -- https://semaphoreui.com/
+>
+> This guide explains how to set up Semaphore UI to use authentik as the OAuth provider for the login to the WebGUI.
+
+## Preparation
+
+The following placeholders will be used:
+
+- `semaphore.company` is the FQDN of the engomo install.
+- `authentik.company` is the FQDN of the authentik install.
+- `ak.cert` is the self-signed certificate that will be used for the service provider.
+
+## Semaphore UI configuration
+
+Create an application and an OAuth2/OpenID provider in authentik. Use the following parameters for the OAuth2/OpenID provider:
+
```

Comment:
Please switch the order of these sections, so that the application part comes first, then the provider, to reflect the order of the Wizard process.